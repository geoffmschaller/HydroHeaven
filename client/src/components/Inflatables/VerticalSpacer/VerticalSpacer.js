import React from 'react';
import PropTypes from 'prop-types';

const VerticalSpacer = (props) => {

	return (
		<div style={{width: 100 + '%', height: props.height + 'px', clear: 'both'}}/>
	);

};

VerticalSpacer.propTypes = {
	height: PropTypes.number.isRequired
};

export default VerticalSpacer;