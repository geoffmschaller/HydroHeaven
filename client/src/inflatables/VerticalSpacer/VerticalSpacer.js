import React from 'react';
import PropTypes from 'prop-types';

const VerticalSpacer = (props) => (
	<div style={{ width: `${100}%`, height: `${props.height}px`, clear: 'both' }} />
);

VerticalSpacer.propTypes = {
	height: PropTypes.number.isRequired,
};

export default VerticalSpacer;
