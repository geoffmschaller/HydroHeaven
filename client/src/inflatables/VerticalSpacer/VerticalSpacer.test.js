import React from 'react';
import VerticalSpacer from "./VerticalSpacer";

const wrapper = <VerticalSpacer height={500}/>;

describe("Vertical Spacer Inflatable", () => {
	it("Renders Correctly", () => {
		expect(wrapper).toMatchSnapshot();
	});
});
