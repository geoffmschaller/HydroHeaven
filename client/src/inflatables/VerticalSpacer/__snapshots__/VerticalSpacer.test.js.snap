// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Vertical Spacer Inflatable 1`] = `
<VerticalSpacer
  height={500}
/>
`;
